### PR TITLE
Add server side handling for [MS-RDPET] + add APIs to get notified about DVC creation status (CreationStatus from DVC response PDU)

### PR DESCRIPTION
**Pull request by pnowack**
This is a forward port of https://github.com/pnowack/FreeRDP/commits/rdpet (rebased on top of stable-2.0). Tested the stable-2.0 with g-r-d with https://gitlab.gnome.org/pnowack/gnome-remote-desktop/-/commits/rdpet

### CODE REVIEW

**Comment by hardening** [comment]
Overall LGTM.
Don't we already have a PubSub notification when channels are opened ?

**Comment by pnowack** [comment]
> Don't we already have a PubSub notification when channels are opened ?

Afaik, this is only the case for the client side, but not the server side.

**Comment by hardening** [comment]
@freerdp-bot can you test again please ?

**Comment by akallabeth** [comment]
@pnowack sorry, looks like my review yesterday was not submitted :/

File: channels/telemetry/server/telemetry_main.c [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
```
[the remark this answers is not in the document]
Comment (pnowack) [reply channels/telemetry/server/telemetry_main.c]: Ah, okay. Didn't noticed it. Fixed.

File: channels/telemetry/server/telemetry_main.c [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+	}
+
+	if (WTSVirtualChannelRead(telemetry->telemetry_channel, 0, (PCHAR)Stream_Buffer(s),
+	                          (ULONG)Stream_Capacity(s), &BytesReturned) == FALSE)
+	{
+		WLog_ERR(TAG, "WTSVirtualChannelRead failed!");
+		goto out;
+	}
+
+	if (Stream_GetRemainingLength(s) < 2)
```
[the remark this answers is not in the document]
Comment (pnowack) [reply channels/telemetry/server/telemetry_main.c]: Done.

**Review by akallabeth — APPROVED** [review]
LGTM

**Comment by akallabeth** [comment]
@freerdp-bot test

**Pull request merged**